Sourcery/Parsing/Composer.swift

```diff
@@ -25,6 +25,17 @@ struct Composer {
         let types = parserResult.types
         let typealiases = self.typealiasesByNames(parserResult)
 
+        //set definedInType for all methods and variables
+        types
+            .forEach { type in
+            type.variables.forEach { variable in
+                variable.definedInType = type
```

Comment:
For typealias `actualDefinedInTypeName` is not resolved properly. See the comment above for proposal.